recipe/meta.yaml

```diff
@@ -1,10 +1,10 @@
 package:
     name: gridutils
-    version: 1.01.0
+    version: v1.0
 
 source:
     git_url: https://github.com/sakov/gridutils-c.git
-    git_tag: master  # upstream does not create releases :-(
```

Comment:
The guy is an old school scientific programmers. I asked for tags/releases and got a few, but I won't insist as this seems to be just a nuisance to him. Next time I will uses commit SHAs.